rero_ils/modules/patrons/jsonschemas/patrons/patron-v0.0.1.json

```diff
@@ -334,6 +336,22 @@
           }
         }
       }
+    },
+    "blocked": {
+      "title": "Blocking",
+      "description": "A patron with a blocked account cannot request and borrow items, but can still renew and check in items.",
+      "type": "boolean"
+    },
+    "blocked_note": {
+      "title": "Reason",
```

Comment:
Why there is such a difference between the key and the title value?

Reply:
PO asked me to change titles. That's why it's so different.
On my side, I know that in some later US, we will change the behaviour, and so "blocked_note" would be a note about the "blocked" functionnality.

Reply:
Ok, thanks.